.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,103 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bugs
+description: Report a bug with the Microsoft build of Go
+title: "import/path: issue title"
```

It seems less likely that people using our build will file issues about specific packages compared to upstream. If there *is* an import path it'd still be nice to include, but I wonder if this might just be confusing.

I don't have a great idea what to do about it other than simplify:

```suggestion
title: "issue title"
```